OTAnalytics/plugin_ui/customtkinter_gui/toplevel_flows.py

```diff
@@ -68,6 +82,8 @@ def _get_widgets(self) -> None:
         self.button_ok = CTkButton(master=self, text="Ok", command=self.close)
 
     def _place_widgets(self) -> None:
+        self.label_id.grid(row=0, column=0, padx=PADX, pady=PADY, sticky=tkinter.E)
+        self.entry_id.grid(row=0, column=1, padx=PADX, pady=PADY, sticky=tkinter.W)
```

Comment:
I propose

- placing the name/id label+entry below the two section dropdowns and
- adding an `_update_entry_name/id` method to update the section name by pattern `<start_name> --> <end_name>` when one of the sections is changed (alternatively, we could add a button to "autofill" the name to prevent overwriting custom names entered previously)

Reply:
@briemla just let me know, if I can help. We can also postpone this issue

Reply:
I will address this in OP#2125